### ISSUE TITLE
Scream 3.3 - cannot mute device in windows

### ISSUE THREAD

**Issue by solarsea** [duncanthrax/scream#49]
Thanks for the mono fix for 3.3, I've confirmed that it works by switching it on and observing the throughput in my router go down in half as opposed to stereo.

In the volume mixer I cannot mute it now via the mute/unmute button. Even when muted scream continues to play.

Volume control works.

**Comment by martinellimarco** [comment]
That's odd, the mono fix was literally a one line fix that have nothing to do with mute / unmute. I can't test this now, I'll see it in 8 hours when I'll be back at home.

In the meantime, can you tell me if the problem is only with mono or if you can't mute it in stereo too?

Thanks.

**Comment by duncanthrax** [comment]
Also, when you say "continues to play", do you mean "continues to send data at zero volume"?

**Comment by solarsea** [comment]
It continues to send data at whatever level was the current volume before muting. The mute toggle only greys out the volume bars in the volume control/mixer window and the toggles the muted/unmuted icon in the icon tray.

**Comment by duncanthrax** [comment]
Muting or setting volume to zero does not stop the stream, you do need to stop any playback from occuring.

**Comment by solarsea** [comment]
That's true. Hardware devices can be muted though, even with the apps playing - I thought scream should be able to do it too. No worries if it doesn't, just pointing it out.
